src/styles/components/_code-pattern.scss

```diff
@@ -19,12 +19,16 @@
     grid-template-columns: minmax(0, 1fr);
 
     @include bp(md) {
-      grid-template-columns: minmax(0, 1fr) minmax(0, 1fr);
+      grid-template-columns: 1fr 1fr;
```

Comment:
Can we keep the old code commented to revert in M94? And open an issue to revert?